minCostFlow.py

```diff
@@ -158,7 +158,9 @@ def main(args):
                         required=True)
     parser.add_argument('--flow',
                         help='The amount of flow pushed through the network.',
-                        type=float)
+                        type=float,
+                        required=False,
+                        default=1.0)
```

Comment:
```suggestion
                        type=int,
                        required=False,
                        default=1)
```
I noticed the `min_cost_flow` requires `flow` to be an `int` so let's enforce that here.